packages/esm-patient-common-lib/src/workspaces/workspaces.ts

```diff
@@ -167,6 +167,11 @@ export function launchPatientWorkspace(name: string, additionalProps?: object) {
       });
     }
   }
+  if (store.getState().workspaceWindowState === 'hidden') {
+    updateWorkspaceWindowState(
+      store.getState().openWorkspaces[0].preferredWindowSize === 'maximized' ? 'maximized' : 'normal',
```

Comment:
It really looks very precisely like you are modifying the `preferredWindowSize`, which should be regarded as immutable. I appreciate your very thorough analysis of the requirements but this is not a good solution.

Reply:
I might not be getting your point of mutating the preferred window size. I am just updating the workspace window size to either be maximized or normal. I am not changing the preferred window size of an opened workspace. I am knowing whether the workspace to opened in full screen ora normal size screen

Reply:
Wow, I am an idiot. Sorry @vasharma05 . You're completely right.

Reply:
No worries @brandones !
Thanks for the review!